database/migrations/2019_02_11_222440_create_arts_table.php

```diff
@@ -16,7 +16,9 @@ public function up()
         Schema::create('arts', function (Blueprint $table) {
             $table->increments('id');
             $table->integer('artist_id')->unsigned();
-            //TODO TM!!!!!!!!!!!!
+            $table->string('title')->nullable();
+            $table->date('publish')->nullable(); //Ferniseringsdato (ikke oprettelsesdato på Wign)
+            $table->bigInteger('views')->unsigned()->default(0);
```

Comment:
En anden mulighed er at lave en many-to-many relationship mellem arts og users - så hver gang en user har set en art, laver vi en relation på (Eller incrementere tallet af "views" der er på netop den relationship) - så vi kan altid sørge for at sikre os at brugeren ALTID ser en af de arts de mindst har set.

Reply:
Den er ikke forretningskritisk funktion, hvorfor jeg tænker at det er ikke super nødvendigt at implementere en fancy spredning af tilfældighed.

Reply:
Iorden! Hvad med at føje dette til backlog så vi ikke glemmer det?